images/runtime/commonbase/buster.Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM debian:buster-slim
```

Comment:
>debian:buster-slim [](start = 5, length = 18)

I see that the stretch version of this base image is deriving from `buildpack-deps:stretch-curl`, but here we are not doing the same. Why? is it because of the size?